Source/GoldenRetriever.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+public struct GoldenRetriever {
+
+  public func fetch(resource: String, closure: (data: NSData!, error: NSError!) -> Void) {
```

Comment:
Is there a reason to use an Implicitly Unwrapped Optional? I try to use it as less as possible.